### PR TITLE
Replace Android system Toast

### PR DESCRIPTION
**Pull request by dg0yt**
Fix GH-1305 by own implementation.

### CODE REVIEW

**Comment by dg0yt** [comment]
@lpechacek  I asked for review, but I hope it is okay if I merge this tonight anyway. I want to quickly create a new dev preview release for Android which fixes the two biggest show stoppers for completing v0.9.0.
Oh, and I have to solve merge conflict.

**Comment by lpechacek** [comment]
Apologies for delayed response. I'm on vacation with limited Internet access and while I've noticed the patch set, I missed the review request.

I glanced at the patches and they look good to me. I've noticed only one nit but that's not a blocker for merge, which has happened already.

File: src/gui/widgets/toast.h [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+public:
+	/**
+	 * Constructs a new Toast.
+	 */
+	Toast(QWidget* parent = nullptr);
+	
+	/**
+	 * Shows a Toast.
+	 * 
+	 * @param region   The global region of interest. The Toast will be placed
```
Comment (lpechacek) [inline src/gui/widgets/toast.h]: `showText()` does not have region parameter. But I think Doxygen can deal with the extra line.

Comment (dg0yt) [reply src/gui/widgets/toast.h]: Thanks!

**Pull request merged**